tests/test_view/test_views.py

```diff
@@ -460,51 +324,62 @@ def test_create_tokens(client: TestClient, token: str):
     j = resp.json()
     assert j["permissions"] == ["store_management:2"]
     # Limited token can't access higher scopes
-    assert (
-        client.post(
-            "/token",
-            json={"permissions": ["store_management"]},
-            headers={"Authorization": f"Bearer {j['id']}"},
-        ).status_code
-        == 403
+    resp = client.post(
+        "/token",
+        json={"permissions": ["store_management"]},
+        headers={"Authorization": f"Bearer {limited_token}"},
     )
-
-    assert client.post("/users", json=LIMITED_USER_DATA).status_code == 422  # Already created
+    assert resp.status_code == 403
     # Strict mode: non-superuser user can't create superuser token
-    assert client.post("/token", json={**LIMITED_USER_DATA, "permissions": ["server_management"]}).status_code == 422
+    resp = client.post(
+        "/token",
+        json={"email": limited_user["email"], "password": limited_user["password"], "permissions": ["server_management"]},
+    )
+    assert resp.status_code == 422
     # Non-strict mode: silently removes server_management permission
     resp = client.post(
         "/token",
-        json={**LIMITED_USER_DATA, "permissions": ["server_management"], "strict": False},
+        json={
+            "email": limited_user["email"],
+            "password": limited_user["password"],
```

Comment:
As for this, we can just move password to constants and avoid exposing it to outside